### DISCUSSION TITLE
How to draw a table with data with PdfPig?

### DISCUSSION THREAD

**Discussion by PlashSpeed-Aiman** [UglyToad/PdfPig#927] [General]
Greetings,

I would like to ask how to draw a table using this library. I have a set of data that need to be put inside a table and export to PDF. If you could kindly prescribe me any strategies to overcome this problem, that would be appreciated.

**Comment by kamilk91** [comment]
Id like to up this issure, because have big need for drawing table.

**Comment by changetocoding** [comment]
Checkout https://github.com/BobLd/tabula-sharp or https://github.com/BobLd/camelot-sharp

I've update the wiki with the information:
https://github.com/UglyToad/PdfPig/wiki

**Reply by markat1** [reply]
@changetocoding hmm, could you provide an example how to draw a table? links you mentioned like tabula-sharp only extract data from table, but can't be used for creating tables? right?
